### PR TITLE
Teraz gramy button

### PR DESCRIPTION
**Pull request by tymwitko**
Added functionality to the "Teraz gramy" button. The main icon acts the same way as the main page player button, thus toggling the current radio stream. Below the icon there's a "Teraz gramy" widget title with the stream's description (RDS) below
Screenshot_20240719_211427
Screenshot_20240719_211433

### CODE REVIEW

File: mobile/lib/components/teraz_gramy/teraz_gramy_widget.dart [hunk 1]
```diff
@@ -20,40 +23,89 @@ class TerazGramyWidget extends StatelessWidget {
...
+              StreamTitle(
+                audioHandler: audioHandler,
+                width: MediaQuery.of(context).size.width,
                 style: context.textStyles.textMedium.copyWith(
                   height: 1.5,
                 ),
               ),
             ],
-          ),
+          },
```
Comment (julianuziemblo) [inline mobile/lib/components/teraz_gramy/teraz_gramy_widget.dart:81]: Also here - after the aforementioned correction, both versions have the same text above - "Teraz gramy", so this should really only be a switch between the Text('No stream title') and the StreamTitle widgets, kinda like so:

```dart
    children: [
        Text(
            'Teraz gramy',
             style: context.textStyles.textPlayer.copyWith(
               color: context.colors.highlightGreen,
             ),
           ),
           switch (audioHandler) {
             null => Text(
                 'No stream title',
                 style: context.textStyles.textMedium.copyWith(
                   height: 1.5,
                 ),
               ),
             _ => StreamTitle(
                 audioHandler: audioHandler,
                 width: MediaQuery.of(context).size.width,
                 style: context.textStyles.textMedium.copyWith(
                   height: 1.5,
                 ),
               ),
           },
         ],
```

**Review by julianuziemblo — CHANGES_REQUESTED** [review]
Works okay, there is an issue with the button being centered. Also, there is a lot of unnecessary code, which makes this much less readable, should be refactored.

**Comment by Tombiczek** [comment]
It seems like it doesn't work on iOS. I get:
```
flutter: stream name:
flutter: stream title:
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: (-11850) Operation Stopped
#0      AudioPlayer._load (package:just_audio/just_audio.dart:870:9)
<asynchronous suspension>
#1      AudioPlayer._setPlatformActive.setPlatform (package:just_audio/just_audio.dart:1494:28)
<asynchronous suspension>
flutter: stream name:
flutter: stream title:
[ERROR:flutter/impeller/base/validation.cc(59)] Break on 'ImpellerValidationBreak' to inspect point of failure: Requested mip_count 4 exceeds maximum supported for size (4, 5)
flutter: stream name:
flutter: stream title:
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: (-11850) Operation Stopped
#0      AudioPlayer._load (package:just_audio/just_audio.dart:870:9)
<asynchronous suspension>
#1      AudioPlayer._setPlatformActive.setPlatform (package:just_audio/just_audio.dart:1494:28)
<asynchronous suspension>
```

Either playing the radio from the `Teraz gramy` button as well as the bottom bar does not work.

**Review by Kszesiek — CHANGES_REQUESTED** [review]
Minor nitpicks, good job overall

**Review by Kszesiek — APPROVED** [review]
Approve

**Review by julianuziemblo — APPROVED** [review]
LGTM! (_of course..._)

**Pull request merged**